.test-infra/jenkins/job_beam_PostRelease_NightlySnapshot.groovy

```diff
@@ -42,11 +42,21 @@ job('beam_PostRelease_NightlySnapshot') {
   common_job_properties.setPostCommit(
       delegate,
       '0 11 * * *',
-      false,
-      'dev@beam.apache.org')
+      false)
+
+
+  // Allows triggering this build against pull requests.
+  common_job_properties.enablePhraseTriggeringFromPullRequest(
+      delegate,
+      './gradlew :release:runQuickstartJavaDirectNightly',
+      'Run Dataflow PostRelease')
 
   steps {
-    // Run a quickstart from https://beam.apache.org/get-started/quickstart-java/
-    shell('cd ' + common_job_properties.checkoutDir + '/release && groovy quickstart-java-direct.groovy')
+    // Run a quickstart from https://beam.apache.org/get-started/quickstart-java
+    gradle {
+      rootBuildScriptDir(common_job_properties.checkoutDir)
+      tasks(':release:runQuickstartJavaDirectNightly')
```

Comment:
Shouldn't we attempt to run all of them by having one task defined in `release/build.gradle` that depends on all the quickstarts?

Reply:
Done. Changed to have a single task, :release:runQuickstartsJava